### ISSUE TITLE
Eater of the Dead (New Card)

### ISSUE THREAD

**Issue by kelsioboy** [magarena/magarena#1178]
name=Eater of the Dead
image=http://magiccards.info/scans/en/med/67.jpg
value=3.400
rarity=U
type=Creature
subtype=Horror
cost={4}{B}
pt=3/4
ability={0}: Exile target creature card from a graveyard.~and untap SN. Activate this ability only if SN is tapped.
timing=main
oracle={0}: If Eater of the Dead is tapped, exile target creature card from a graveyard and untap Eater of the Dead.

**Comment by ShawnieBoy** [comment]
This is a problem card due to some pretty technical mtg rules. The effect should check the state of Eater of the Dead before exiling a card. With this interpretation, you can still activate it multiple times before effects resolve and still exile more than one card, which shouldn't be possible.

**Comment by kelsioboy** [comment]
What if I put it to activate the effect only once ?

**Comment by melvinzhang** [comment]
@kelsioboy putting the "If" condition in front is supported, I've made a two tweaks to your card script.

1) add back the "If" condition in front

2) add an AI limitation to prevent endless activation

See the linked commit for the details. Thanks!
